src/pages/Contact.tsx

```
@@ -1,15 +1,71 @@
+}
+
+console.log("model",model);
+try {
+  const response = await contact({data:{payload:model}}).unwrap();
+  if(response?.status){
+    toast.success(response.message)
+  } else{
+    toast.error(response.message)
+  }
+    
+} catch (error:any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)